config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  # API endpoints
+  namespace :api do
+    resources :users, param: :username, only: [:show, :create]
+  end
```

Comment:
@AndreaM2429 I kindly suggest you remove the routes from this implementation as it belongs to another task.

Reply:
@NitBravoA92 thanks for the observation, I forgot that it belongs to another task.

Reply:
@AndreaM2429 No problem. Good job applying the suggested changes!